xarray/tests/__init__.py

```diff
@@ -114,6 +114,18 @@ def requires_bottleneck(test):
     return test if has_bottleneck else pytest.mark.skip('requires bottleneck')(test)
 
 
+flakey = pytest.mark.skipif(
+    pytest.config.getoption("--skip-flakey"),
+    reason="set --skip-flakey option to skip flakey tests"
+)
+
+
+slow = pytest.mark.skipif(
+    not pytest.config.getoption("--run-slow"),
+    reason="set --run-slow option to run slow tests"
```

Comment:
Just to be clear, it looks like the current default behavior is:
- Don't skip flakey tests
- Skip slow tests

I think we actually want to flip these. Consider someone running our test suite for the first time. They probably want to understand if their xarray installation worked. It's not a big deal if this is a little slow, because they probably aren't doing this many times in a row, but not testing part of the test suite would be unfortunate. Once they notice the tests are slow, they will probably quickly look for a `--skip-slow` option.